go/billing/api.py

```diff
@@ -228,34 +244,49 @@ def create_transaction_interaction(self, cursor, account_number,
         session_credits = MessageCost.calculate_session_credit_cost(
             session_cost, markup_percent)
 
+        session_unit_credits = MessageCost.calculate_session_unit_credit_cost(
+            session_unit_cost, markup_percent)
```

Comment:
Ya, their isn't any transaction cost associated to it, since the unit cost is a preliminary step for working out the session length cost. We have a corresponding credit field for every other cost field, so I added it for the sake of completeness. It might also be useful for auditing purposes, though that's a bit of a guess. Not really against removing it, but wondering if there is any reason to remove it. I guess we'd be storing slightly less things, not sure if that's important. Any other reasons?